.github/workflows/helm.yml

```diff
@@ -17,112 +17,7 @@ concurrency:
-
-          if [ "${RES}" == "No results found" ]; then
-            echo "Version incremented"
-          else
-            echo "Validation failed, you should upgrade the chart version in Chart.yaml"
-            exit 1
-          fi
-
-  generate-chart-readme:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout bitnami-labs/readme-generator-for-helm
-        uses: actions/checkout@v4
-        with:
-          repository: "bitnami-labs/readme-generator-for-helm"
-          ref: "2.5.0"
-          path: readme-generator-for-helm
-
-      - name: Install readme-generator-for-helm dependencies
-        run: cd readme-generator-for-helm && npm install
-
-      - uses: actions/checkout@v4
-        with:
-          path: backend
-
-      - name: Execute readme-generator-for-helm
-        run: readme-generator-for-helm/bin/index.js -r backend/charts/substra-backend/README.md -v backend/charts/substra-backend/values.yaml
-
-      - name: Check diff
-        run: |
-          cd backend/
-          if [ -z "$(git status --porcelain)" ]; then
-            exit 0
-          else
-            echo "There should be no change generated, please run 'make doc' in backend/charts/ to update the chart README.md"
-            git diff
-            exit 1
-          fi
-
-  publish:
-    name: Publish
-    runs-on: ubuntu-latest
-    if: github.ref == 'refs/heads/main' && github.event_name == 'push'
-    needs:
-      - test
-      - generate-chart-readme
-    steps:
-      - uses: actions/checkout@v4
-
-      - uses: azure/setup-helm@v3.5
-        with:
-          version: "v3.12.0"
-        id: install
-
-      - name: Add dependencies repo
-        run: |
-          helm repo add bitnami https://charts.bitnami.com/bitnami
-          helm repo add std-helm https://charts.helm.sh/stable
-          helm repo add twuni https://helm.twun.io
-
-      - name: Package chart
-        run: |
-          helm dep build charts/substra-backend
-          helm package charts/substra-backend
-
-      - name: Clone Substra charts
-        uses: actions/checkout@v4
-        with:
-          repository: Substra/charts
-          ref: "main"
-          token: ${{ secrets.CHARTS_GITHUB_TOKEN }}
-          path: substra-charts
-
-      - name: Publish chart
-        run: |
-          mv substra-backend-$(grep -e "^version" charts/substra-backend/Chart.yaml | cut -c10-).tgz substra-charts/
-          cd substra-charts
-          helm repo index .
-          git add .
-          git config --global user.email "gh-actions@github.com"
-          git config --global user.name "GitHub Action"
-          git commit -s --message "GitHub Action: ${{ github.repository }}@${{ github.sha }}"
-          git push --quiet --set-upstream origin main
+  test-generate-publish:
+      uses: substra/substra-gha-workflows/.github/workflows/helm.yml@chore/check-chart-lock-changes
```

Comment:
Is it normal we are targeting the `chore/check-chart-lock-changes` rather than a long-lived branch or tag?

Reply:
You're right! Just changed it